geocoding_platform_interface/pubspec.yaml

```
@@ -3,22 +3,22 @@ description: A common platform interface for the geocoding plugin.
 homepage: https://github.com/baseflow/flutter-geocoding/tree/master/geocoding_platform_interface
 # NOTE: We strongly prefer non-breaking changes, even at the expense of a
 # less-clean API. See https://flutter.dev/go/platform-interface-breaking-changes
-version: 1.0.1+1
+version: 2.0.0+2
```

I think we can bump this to version `2.0.0` (no need for the extra patch nummer):
```suggestion
version: 2.0.0
```